Source/Utils/WarpXUtil.cpp

```diff
@@ -213,6 +213,86 @@ WarpXParser makeParser (std::string const& parse_function, std::vector<std::stri
     return parser;
 }
 
+/**
+ * \brief Ensures that the blocks are setup correctly for the RZ spectral solver
+ * When using the RZ spectral solver, the Hankel transform cannot be
+ * divided among multiple blocks. Each block must extend over the
+ * entire radial extent.
+ * The grid can be divided up along z, but the number of blocks
+ * must be >= the number of processors.
```

Comment:
In the Sphinx documentation for the parameters `max_grid_size` and `blocking_factor`, could you mention the constraints for RZ spectral, and mention that WarpX will automatically modify the input values so as to satisfy them?

Reply:
Yes, I can add the documentation.

Reply:
Done